src/components/Forms/ClientInfo.js

```diff
@@ -10,8 +11,75 @@ import MedicalHistory from "./MedicalHistory";
 import PhoneInput from "./PhoneInput";
 import { useTranslation } from "react-i18next";
 
+const genders = ["Male", "Female", "Not Specific"];
+
 export default function FormPropsTextFields() {
   const { t } = useTranslation();
+  const [firstName, setFirstName] = useState("");
```

Comment:
This works! But I think a better way to implement this is to define a DTO class called `User`. 
So you will only need one useState(defaultUser).

Reply:
That is a good idea! I plan to refactor the code later